design/workitems.go

```diff
@@ -101,9 +101,8 @@ var workItemReorder = Reorder(
 	workItem,
 	position)
 
-// new version of "list" for migration
+// endpoints that DO NOT depend on the space id (ie, when the work item ID is specified in the URL, there's no need to pass the space ID)
 var _ = a.Resource("workitem", func() {
-	a.Parent("space")
 	a.BasePath("/workitems")
 	a.Action("show", func() {
```

Comment:
It's a good test to see if they are following the links as provided by the Spaces API ;)